<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestStatWidget</class>
 <widget class="QWidget" name="TestStatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测试统计</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="_testFilterLabel">
       <property name="text">
        <string>测试筛选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="_testFilterLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="_testStatListView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="_exportPushButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>导出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
